pkg/asset/installconfig/openstack/validation/cloudinfo.go

```diff
@@ -123,3 +131,34 @@ func (ci *CloudInfo) getNetwork(networkName string) (*networks.Network, error) {
 
 	return network, nil
 }
+
+func (ci *CloudInfo) getFloatingIPs(api string, ingress string) (API *floatingips.FloatingIP, Ingress *floatingips.FloatingIP, err error) {
+	API = &floatingips.FloatingIP{}
+	Ingress = &floatingips.FloatingIP{}
+
+	if api != "" {
+		fip, err := floatingips.Get(ci.clients.networkClient, api).Extract()
+		if err != nil {
+			var gerr gophercloud.ErrDefault404
+			if !errors.As(err, &gerr) {
```

Comment:
This comparison is wrong, it should be:
```suggestion
			if errors.As(err, &gerr) {
```